SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -736,6 +736,12 @@ int32 USpatialNetDriver::ServerReplicateActors_ProcessPrioritizedActors(UNetConn
 				// or it's an editor placed actor and the client hasn't initialized the level it's in
 				if (Channel == nullptr && GuidCache->SupportsObject(Actor->GetClass()) && GuidCache->SupportsObject(Actor->IsNetStartupActor() ? Actor : Actor->GetArchetype()))
 				{
+					if (!ClassInfoManager->IsSupportedClass(Actor->GetClass()))
```

Comment:
Shouldn't we not be generating schema either then in this case? I don't think we respect the `NotSpatial` flag there as well.

Reply:
We do skip it. There is a problem at the moment, however, where the generated schema isn't removed when you make a class `NotSpatial` and regenerate schema, you have to delete SchemaDatabase and then generate (https://improbableio.atlassian.net/browse/UNR-997)

Reply:
Ah ok. Also I think it the latest version of master, there is no longer a `checkf` in SpatialClassInfoManager, we simply just stop the game (a request from Darewise).

Reply:
Right. Which would still be pretty bad since the class is explicitly marked `NotSpatial`.